### PR TITLE
Run benchmarks, push data to tinybird and gh-pages

### PR DESCRIPTION
**Pull request by gsoltis**
New revision of the large monorepo benchmark. Still set to manual. Produces both benchmark-action-compatible json as well as tinybird-compatible ndjson.

Sends ndjson to tinybird and json to gh-pages.

Tested on my own fork of the repo.

### CODE REVIEW

File: benchmark/package.json [hunk 1]
```diff
@@ -4,12 +4,13 @@
   "dependencies": {
```
Comment (tknickman) [inline benchmark/package.json:4]: Maybe doesn't totally fit the mold - but It might be nice to move this to `packages/` in the future

**Review by tknickman — APPROVED** [review]

File: benchmark/package.json (hunk 1 above)
Comment (gsoltis) [reply benchmark/package.json:4]: Yeah, might make sense.

**Review by mehulkar — APPROVED** [review]
Code looks good. Just for my understanding, why do we push to both Tinybird and gh-pages branch?

**Comment by gsoltis** [comment]
@mehulkar the github action keeps a json file up to date and provides a template we can eventually use to make nice graphs. Tinybird is for our own internal analysis

**Pull request merged**